edge-hub/core/test/Microsoft.Azure.Devices.Edge.Hub.E2E.Test/Cloud2DeviceTest.cs

```diff
@@ -247,7 +222,14 @@ async Task Cleanup(DeviceClient deviceClient, ServiceClient serviceClient, Regis
         {
             if (deviceClient != null)
             {
-                await deviceClient.CloseAsync();
+                try
+                {
+                    await deviceClient.CloseAsync();
+                }
+                catch
+                {
+                    // ignore
```

Comment:
Why would this fail? Is it worthwhile to just catch the specific exception(s) or do we just not care after cleanup call? (not saying it should be done differently, just trying to understand :) )

Reply:
it can fail from sdk if it was already closed, I would say we should not fail the test if close fails.